Add migration-issues page to explain Proposed Migration in detail (#121)
ubuntu/ubuntu-project-docs: docs/contributors/advanced/plus-one-maintenance.md

ubuntu-devel list](https://lists.ubuntu.com/archives/ubuntu-devel/), as well as
the [build status report](http://qa.ubuntuwire.com/ftbfs/).

#### Migration issues (aka "update-excuses")

The [update-excuses
page](https://ubuntu-archive-team.ubuntu.com/proposed-migration/update_excuses.html),

> provides a visual checklist of issues including the aforementioned

Reviewer: ```suggestion
provides a visual checklist of issues including
```

> transitions and build failures (FTBFS), along with test failures,

Reviewer: ```suggestion
transitions and build failures (FTBFS), test failures,
```

> dependency mismatches, and assorted other problems.

Reviewer: ```suggestion
dependency mismatches, and other problems.
```

[visual-excuses](https://github.com/mclemenceau/visual-excuses) or [ubuntu-excuses](https://github.com/mclemenceau/ubuntu-excuses) to try (which are also available as snaps) and find
high-impact issues that would unblock large sets of packages.

> In general, every package update uploaded to the distro archive must go

Reviewer: ```suggestion
In general, every package update uploaded to the Archive must go
```

> through a battery of checks while it sits in a staging area called the

Reviewer: ```suggestion
through a series of checks while it sits in a staging area called the
```

"proposed pocket".  Once all the checks have been passed, the package is
permitted to move, or "migrate", from this pocket into the official

> release pocket.  Otherwise, {ref}`migration issues <migration-issue>`

Reviewer: ```suggestion
release pocket. Otherwise, migration issues
```

Reviewer: The same link is provided in the next paragraph

hindering the package may need to be manually reviewed and resolved.

Unfortunately, this update-excuses report is rather impenetrable due

> both to its size and terse jargon.  See {ref}`migration-ssue` for a more

Reviewer: ```suggestion
both to its size and terse jargon. See {ref}`proposed-migration` for a more
```

detailed explanation of the migration process, issues that can arise,

> and how to interpret the ubuntu-excuses report's jargon.

Reviewer: ```suggestion
and how to interpret the ubuntu-excuses report.
```
